src/components/Books.js

```diff
@@ -0,0 +1,25 @@
+import React, { useState } from 'react';
+import Book from './Book';
+import Addbook from './Addbook';
+
+export default function Books() {
+  const [books] = useState([{
+    title: 'react', author: 'karan',
+  }]);
+  return (
+    <div className="bookContainer">
+      <ul className="booklist">
+        {books.map((book) => (
+          <li key={book.title + book.autor}>
+            <Book title={book.title} author={book.author} />
+
+            <button className="remove" type="button">
+              remove
+            </button>
```

Comment:
-There is already a `remove` button in the `Book` component, please remove this button if it is not needed so you have only what is required on your app. Please fix this 😄